UnityProject/Assets/Scripts/Player/Mind.cs

```diff
@@ -262,6 +296,69 @@ public void Ghost()
 		SyncActiveOn(IDActivelyControlling, GetDeepestBody().netId);
 	}
 
+	/// <summary>
+	/// Spawn the ghost for this player and tell the client to switch input / camera to it
+	/// </summary>
+	[Command]
+	public void CmdSpawnPlayerGhost()
+	{
+		ServerSpawnPlayerGhost();
+	}
+
+	[Server]
+	public void ServerSpawnPlayerGhost(bool skipCheck = false)
+	{
+		//Only force to ghost if the mind belongs in to that body
+		if (skipCheck)
+		{
+			Ghost();
+			return;
+		}
+
+		var Deepest = GetDeepestBody();
```

Comment:
```suggestion
		var deepest = GetDeepestBody();
```
couldn't this just be called ``body``?